src/Application/Contracts/IRulesContext.cs

```diff
@@ -9,6 +8,7 @@ public interface IRulesContext
 {
     DbSet<Rule> Rules { get; set; }
```

Comment:
Just an FYI.  It seems a bit weird to have classes with a suffix of 'Model', 'Result', and some with no suffix at all.

Reply:
I've added a task to https://github.com/SSWConsulting/SSW.Rules.GPT/issues/151 to normalise this as part of the EF config extraction